### ISSUE TITLE
About 'add' aggregator

### ISSUE THREAD

**Issue by jindeok** [rusty1s/pytorch_geometric#2722]
Hello, I really appreciate your kind codes and their comments.

My question is that does the **'add'** aggregator in GNN **normalize** aggregated messages?

**Comment by rusty1s** [comment]
No, `aggr="add"` refers to summing up all incoming messages. As such, there is no normalization applied.
